docs/user-guide/install-zowe-client-side-components.md

```diff
@@ -0,0 +1,21 @@
+# Installing Zowe client-side components
+
+Prepare yourself to install Zowe client-side components.
+
+:::info**Required roles:** systems programmer, security administrator, storage administrator
+:::
```

Comment:
It seems like there is a missing newline for this admonition, which affects how it is rendered:

![image](https://github.com/zowe/docs-site/assets/86500639/7316d0e6-0267-47bf-bf5c-ff6b4a63b902)

vs.:

![image](https://github.com/zowe/docs-site/assets/86500639/12add9b2-04c8-4a21-91a9-9576bdb0a3b7)

I wasn't sure if we preferred this minimal style here, or if we wanted to add the new line to match the other admonition formats:

```suggestion
:::info
**Required roles:** systems programmer, security administrator, storage administrator
:::
```

Reply:
@traeok -- good question. The answer is that this is a new style of admonition that we're using to call out the required roles for a specific article topic. It looks awkward for the admonition title to read "**! INFO REQUIRED ROLES: CAT ONE, KITTY TWO, KITTEN THREE**," so we've played around w/ the syntax to remove the "**INFO**" from the admonition header. It's actually something that UX folks have advised for the API ML content and something I'm now bringing into the CLI content (and, in future PRs, the ZE content)
Good catch, thank ya.